tasks/installation.yml

```diff
@@ -22,3 +22,14 @@
   package:
     name: '{{ docker_packages }}'
     state: present
+
+- name: add cleanup cronjob
```

Comment:
Please add the new variables with default values and a description to defaults/main.yml